lib/hacluster.pm

```diff
@@ -345,6 +345,7 @@ sub check_cluster_state {
 }
 
 # Wait for resources to be started
+# If changing this, remember to also change wait_until_resources_started in tests/publiccloud/sles4sap.pm
```

Comment:
You added this comment, but is there any changes to do in PC code? Or is it "just" to add a note to not forget the PC code?

Reply:
Just to add a note and not forget the PC code.